src/map.rs

```diff
@@ -658,7 +685,7 @@ impl Default for ChunkComponents {
             mesh: Handle::default(),
             material: Handle::default(),
             render_pipeline: RenderPipelines::from_pipelines(vec![RenderPipeline::new(
-                TILE_MAP_PIPELINE_HANDLE.typed()
+                TILE_MAP_PIPELINE_HANDLE.typed(),
```

Comment:
This used to have some lines with PipelineSpecialization -- was that a WASM-specific change?

Reply:
@framp looks like the code I was remembering was your change. I suspect it's not needed anymore.